dll/shellext/mydocs/lang/ro-RO.rc

```diff
@@ -0,0 +1,6 @@
+LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+{
+    IDS_NOSRCFILEFOUND "Fișierul sau folderul sursă '%ls' nu a fost găsit."
```

Comment:
I translated „folder” as „dosar”, and I strongly recommend you to not leave it untranslated (like Microsoft did, unfortunately).